benefits/static/css/styles.css

```diff
@@ -517,26 +533,18 @@ footer .footer-links li a.footer-link:visited {
 }
 
 /* Forms: Text Inputs */
-/* Form Container: Must use .form-container parent to use these styles */
+/* Form Field Container: Must use .form-field-container parent to use these styles */
```

Comment:
Renamed this b/c this div is a container around one form field, rather than the entire form.